processor/src/test/java/org/mapstruct/ap/test/builder/multiple/MultipleBuilderMapperTest.java

```diff
@@ -113,18 +113,7 @@ public void builderMappingMapperConfigDefined() {
         TooManyBuilderCreationMethodsMapper.class
     })
     @ExpectedCompilationOutcome(value = CompilationResult.SUCCEEDED,
-        // We have 2 diagnostics, as we don't do caching of the types, so a type is processed multiple types
         diagnostics = {
-            @Diagnostic(
```

Comment:
So we no longer report this warning?

Reply:
correction: since we only resolve the builder once, we don't report the duplicate anymore :smile:

Reply:
I am not quite following. I think what the test is trying to show that if there are multiple builders (when we try to resolve) we are going to output a warning message. It is not about duplicates, there are 2 potential builder creation methods here.

Reply:
I see it now. There is another `@Diagnostic` with the exact same error